core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/toDataFrame.kt

```diff
@@ -74,6 +74,13 @@ internal val KClass<*>.isValueType: Boolean
             this.isSubclassOf(TemporalAmount::class) ||
             this.isArray
 
+/**
+ * Checks if `KClass` has properties.
+ */
+@PublishedApi
+internal val KClass<*>.hasProperties: Boolean
+    get() = this.memberProperties.isNotEmpty()
```

Comment:
This might cause issues with POJO's, java objects with getters+setters instead of memberProperties

Reply:
added check